test/unit/sessions/session-manager.test.ts

```diff
@@ -1054,4 +1054,24 @@ describe('updateAudioVolume', () => {
         expect(mockSessionHandler.updateAudioVolume).toHaveBeenCalledWith(session, 63);
       });
   });
+
+  describe('updateSessionWithDefaultResolution', () => {
+    it('should call the handler to update session', () => {
+      const session = new MockSession(SessionTypes.collaborateVideo) as any;
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9629161)
```suggestion
      const session = new MockSession(SessionTypes.collaborateVideo) as unknown;
```